tsdb/mapper.go

```diff
@@ -16,8 +16,9 @@ import (
 // aggregate output Time is ignored, and actual Time-Value pairs are contained soley
 // within the Value field.
 type MapperValue struct {
-	Time  int64       `json:"time,omitempty"`  // Ignored for aggregate output.
-	Value interface{} `json:"value,omitempty"` // For aggregate, contains interval time multiple values.
+	Time  int64             `json:"time,omitempty"`  // Ignored for aggregate output.
+	Value interface{}       `json:"value,omitempty"` // For aggregate, contains interval time multiple values.
+	Tags  map[string]string `json:"tags,omitempty"`  // Meta tags for results
```

Comment:
OK, I follow that. Tell me again why you need the tags coming up with MapperValues?